variables.tf

```diff
@@ -120,6 +120,18 @@ variable "alb_authenticate_cognito" {
   default     = {}
 }
 
+variable "alb_enable_deletion_protection" {
+  description = "If true, deletion of the load balancer will be disabled via the AWS API. This will prevent Terraform from deleting the load balancer. Defaults to false."
+  type        = bool
+  default     = null
```

Comment:
These two variables should have default `false` since this is what the ALB module has as defaults - https://github.com/terraform-aws-modules/terraform-aws-alb/blob/master/variables.tf#L7-L17

Btw, description is correctly saying `false` :)

If we change this to `null` it will require an update for everyone using this module.

Reply:
the ALB module makes assumptions by setting `false` as default, but the underlying provider/API ultimately defaults to `false` as well. I was trying to avoid making any assumptions with defaults if they are not required:
<img width="652" alt="image" src="https://user-images.githubusercontent.com/10913471/114461981-70908180-9bb0-11eb-9627-c384b416c0e7.png">
<img width="658" alt="image" src="https://user-images.githubusercontent.com/10913471/114462085-91f16d80-9bb0-11eb-8a6a-ce03102ec6e4.png">

Reply:
@antonbabenko  Personally, I can go either way. However, what @bryantbiggs suggested made sense to me in that setting the values to null is relying on AWS to provide these reasonable or possible preferable defaults. Setting them to default as false in this modules opinionates it further (which may or may not be preferred). 

I'm not entirely clear on how setting this to null will require everyone to update, but that does not sound like a good time.

Like I mentioned, I can go either way. I have only been using this module a couple weeks so I defer to the maintainers judgement. Please tell me what changes to make and I will make them. Thanks

Reply:
I misread all this. You are right, it should be `null`, my bad 👍  Merging right away!